team_admin/discord.md

```diff
@@ -11,7 +11,7 @@ If you do not wish to, or are unable to join Discord, you can still get support
 
 ## Joining
 
-Invite links will be sent to team leaders. Only competitors should be given access.
+Invite links will be sent to team leaders which will ensure you're added to the correct team. Only team members should be given access.
```

Comment:
It might be worth mentioning here (in place of or possibly addition to where we say it below) that where there are multiple teams per school there will be a channel per team.